CompanyWorkActivities.md

```diff
@@ -0,0 +1,54 @@
+Company Work Activities
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+The Company Work Activities service provides a list of work activities associated with a 
+`company_id` and `carotene_id` pair or `onet_id`. If both `carotene_id` and `onet_id` are supplied,
+then the service first attempts to return work activities for the `carotene_id`. If none are 
+found then it falls back to using the `onet_id`. Company Work Activities is available at 
+`/core/company/workactivities`.
```

Comment:
is that first sentence's logic out of date now? and maybe add "is available at [...] in the US, EU, and Staging regions" at the end (since not all APIs are available in all regions, mainly EU)?

Reply:
also, we should probably call out which ONet/Carotene taxonomy version(s) this service supports (again, maybe the Versioning section is best for this). it would be a bad experience for the user to submit, say, a valid caroteneV1 ID and get no results because that ID doesn't exist in caroteneV3.

Reply:
Changed